### DISCUSSION TITLE
Rising Water?

### DISCUSSION THREAD

**Discussion by Kulicee** [Pryaxis/TShock#1730] [General]
What would be the easiest/most efficient way to make water rise in a specific a region(ie. Make water rise by 1 block every 3 seconds)? Is there a specific plugin I could use to do this? Thanks!

**Comment by QuiCM** [comment]
AFAIK there's no plugin that does this currently.
Hypothetically you could write a plugin that spawns water in the region to make it rise

**Comment by Kojirremer** [comment]
The easiest way is to just use a vanilla water generator engine.
https://terraria.fandom.com/wiki/Generator
If you *really* want to use plugins instead, you could use WorldEdit in tandem with a plugin that can run conmands every x seconds, like Essentials+ or AutoBroadcast:
You set it up so that it "floods" a single tile over the area you want receiving water, then settle it (since spawning water with WorldEdit fixes it in place until liquids settle or a tile touching its borders change).

**Reply by Kulicee** [reply]
The second option worked perfectly for what I was doing. Thanks!
